README.md

```diff
@@ -3,3 +3,99 @@
 [![](https://images.microbadger.com/badges/image/adfinissygroup/debug.svg)](https://microbadger.com/images/adfinissygroup/debug "Get your own image badge on microbadger.com") [![](https://images.microbadger.com/badges/version/adfinissygroup/debug.svg)](https://microbadger.com/images/adfinissygroup/debug "Get your own version badge on microbadger.com")
 
 A container image packed with debugging tools
+
+# Quick start
+
+```
+podman run -it --entrypoint=/bin/bash adfinissygroup/debug
+Trying to pull docker.io/adfinissygroup/debug...
+Getting image source signatures
+Copying blob df20fa9351a1 skipped: already exists
+Copying blob 95af1e08e02d done
+Copying config 533c2b3bb9 done
+Writing manifest to image destination
+Storing signatures
+bash-5.0# which tcpdump
+/usr/sbin/tcpdump
+bash-5.0# which nc
+/usr/bin/nc
+bash-5.0# which ping
+/bin/ping
+bash-5.0# which dig
+/usr/bin/dig
+```
+
+# using the debug container on k8s
+
+Modify your deployment / pod to include a second container:
```

Comment:
We can describe this modification of the deployment using a [patch file](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/):

```yaml
# debug-patch.yml
---

spec:
  template:
    spec:
      containers:
      - name: debug
        image: adfinissygroup/debug:latest
```

Apply with: `kubectl patch deployment web --patch "$(cat debug-patch.yml)"`

We could add this as an additional help text to quickly patch existing deployments?

Reply:
gonna open an issue for this